datasets/me/acp_peps/crawler.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise, OccupancyStatus
+from zavod.shed.trans import (
+    apply_translit_full_name,
+    make_position_translation_prompt,
+)
+
+TRANSLIT_OUTPUT = {
+    "eng": ("Latin", "English"),
+}
+POSITION_PROMPT = prompt = make_position_translation_prompt("cnr")
+
+
+def get_latest_date(dates):
+    if dates:
+        # Convert string dates to datetime objects
+        dates = [datetime.strptime(item["datum"], "%Y-%m-%d") for item in dates]
+        latest_date = max(dates)
+        return latest_date.strftime("%Y-%m-%d")
+    return None
+
+
+def crawl_person(context: Context, person):
+    name = person.pop("imeIPrezime")
+    position = person.pop("nazivFunkcije")
+    dates = person.pop("izvjestajImovine")
+    latest_date = get_latest_date(dates)
+
+    entity = context.make("Person")
+    entity.id = context.make_id(name, position)
+    entity.add("name", name)
+    entity.add("topics", "role.pep")
+    for pos in position:
+        position = h.make_position(
+            context,
+            name=pos,
+            country="ME",
+            # lang="cnr",
+        )
+        entity.add("position", pos)
+
+        apply_translit_full_name(
+            context, position, "slk", pos, TRANSLIT_OUTPUT, POSITION_PROMPT
+        )
+
+        categorisation = categorise(context, position, is_pep=True)
+        if not categorisation.is_pep:
+            return
+
+        occupancy = h.make_occupancy(
+            context,
+            entity,
+            position,
+            no_end_implies_current=False,
+            categorisation=categorisation,
+            status=OccupancyStatus.UNKNOWN,
+        )
+        occupancy.add("date", latest_date)
+
+        context.emit(position)
+        context.emit(occupancy)
+
+    context.emit(entity, target=True)
```

Comment:
only emit the person if there was an occupancy for them, and let make_occupancy add the topic. Even though the current implementation will always make an occupancy, it's good to still follow the pattern  because if we change the implementation later to use start/end dates (which means make_occupancy decides whether they're still a PEP), the logic for deciding whether to emit the person is still valid.